javascript/js-rails/rails_backend.md

```diff
@@ -63,11 +65,11 @@ This section contains helpful links to other content. It isn't required, so cons
 
 #### With Ruby on Rails
 
-* Refresher -- how do you set up a basic Rails app?
-* Refresher -- how do you load custom Javascript in a given Rails view page?
-* How does "unobtrusive Javascript" work?
-* How can you pass data from your Rails app to your Javascript?
-* Why would you want to use AJAX to load large batches of data?
+* Refresher: <a class="knowledge-check-link" href="https://www.theodinproject.com/paths/full-stack-ruby-on-rails/courses/ruby-on-rails/lessons/your-first-rails-application-ruby-on-rails#your-first-rails-app" target="_blank"> How do you set up a basic Rails app? </a>
```

Comment:
Are we linking this somewhere in this lesson? I may just not be seeing it.

Reply:
No. I will most likey remove this knowledge check after reviewing my changes again.